virttest/virsh.py

```diff
@@ -4209,7 +4209,9 @@ def vol_upload(name, dfile, options="", **dargs):
     return command(cmd, **dargs)
 
 
-def blkiotune(name, weight=None, device_weights=None, options=None, **dargs):
+def blkiotune(name, weight=None, device_weights=None, device_read_iops_sec=None,
```

Comment:
please update docstring for the parameters.

Reply:
added, thx

Reply:
@lento-sun, your updates are about params in 'blkdeviotune' but not in 'blkiotune'.

Reply:
oops~
updated. The blkdeviotune() has much less available params than virsh api does, need to be updated later. not covered here in cgroup related fix.